otter-sec/anchor
# Add option for jest test scaffolding

I typically always manually change tests from `mocha` to `jest` when starting a new project. As `jest` is a commonly used testing framework, I added a scaffolding option, `--jest`, which scaffolds tests as jest tests instead of mocha. I kept this as a flag so that the prior behavior to create `mocha` tests by default was maintained.

How to test:
1. Build the CLI by running `make build-cli` from repo root
2. Use the newly built cli to create a project (TS and JS); `./anchor/cli/npm-package/anchor init test-ts --jest` and `./anchor/cli/npm-package/anchor init test-js --jest --javascript`
3. `cd test-ts`
4. `anchor test`  and observe success
5. `cd ../test-js`
6. `anchor test` and observe success
